jlayer/src/javazoom/jl/player/JavaSoundAudioDevice.java

```diff
@@ -87,46 +98,51 @@ public void open(AudioFormat fmt) throws JavaLayerException
 	}
 
 	protected void openImpl()
-		throws JavaLayerException
+			throws JavaLayerException
 	{
 	}
 
 
 	// createSource fix.
 	protected void createSource() throws JavaLayerException
-    {
-        Throwable t = null;
-        try
-        {
-			Line line = AudioSystem.getLine(getSourceLineInfo());

```

Comment:
Less intrusive approach would be extracting this line into protected method, and let the application override it, e. g. with with this mixer approach. With smaller change we allow for much greater flexibility.